docs/guides/src/main/server/containers.adoc

```diff
@@ -100,8 +109,10 @@ podman|docker run --name keycloak_auto_build -p 8080:8080 \
         --https-key-store-file=<file> --https-key-store-password=<password>
 ----
 
-Running the above will spin up a Keycloak server which detects and applies the build configuration first. In the example, it's  `--db=postgres --features=token-exchange` to set the used database vendor to postgres and enable the token exchange feature.
+Running this command starts a Keycloak server that detects and applies the build configuration first.
```

Comment:
```suggestion
Running this command starts a Keycloak server that detects and applies the build options first.
```
See #9709 for details about the general naming. "Build configuration" might be a bit ambiguous here.